doc/source/themes/nature_with_gtoc/static/nature.css_t

```diff
@@ -299,25 +299,33 @@ td.field-body blockquote {
     padding-left: 30px;
 }
 
-.rendered_html table {
+// Adapted from the new Jupyter notebook style
+// https://github.com/jupyter/notebook/blob/c8841b68c4c0739bbee1291e0214771f24194079/notebook/static/notebook/less/renderedhtml.less#L59
+table {
     margin-left: auto;
     margin-right: auto;
-    border-right: 1px solid #cbcbcb;
-    border-bottom: 1px solid #cbcbcb;
-}
-
-.rendered_html td, th {
-    border-left: 1px solid #cbcbcb;
-    border-top: 1px solid #cbcbcb;
-    margin: 0;
-    padding: 0.5em .75em;
-}
-
-/**
- * See also
- */
-
-div.seealso dd {
-    margin-top: 0;
-    margin-bottom: 0;
```

Comment:
Mmm probably an accident on my part. I'll push a fix (the rendered_html removal was intentional).